openedx/core/djangoapps/plugins/constants.py

```diff
@@ -2,14 +2,6 @@
 Constants used by django app plugins
 """
 
-# expose constants from edx-django-utils so plugins using these continue to work
```

Comment:
These imports look tricky. Maybe you should keep these.

Reply:
Okay